deeplearning4j/deeplearning4j-modelimport/src/main/java/org/deeplearning4j/nn/modelimport/keras/utils/KerasLayerUtils.java

```diff
@@ -227,6 +227,8 @@ public static KerasLayer getKerasLayerFromConfig(Map<String, Object> layerConfig
             layer = new KerasConvolution2D(layerConfig, enforceTrainingConfig);
         } else if (layerClassName.equals(conf.getLAYER_CLASS_NAME_DECONVOLUTION_2D())) {
             layer = new KerasDeconvolution2D(layerConfig, enforceTrainingConfig);
+        } else if (layerClassName.equals(conf.getLAYER_CLASS_NAME_DECONVOLUTION_3D())) {
+            layer = new KerasDeconvolution3D(layerConfig, enforceTrainingConfig);
```

Comment:
I've always wondered, why that pattern? Why not a public final property? 

Not really a blocker for the PR itself, but I always stumble when I see this.

Reply:
@treo More just legacy. It was originally written by Max. I intend on rewriting this eventually, but for now will just keep it consistent.